.travis.yml

```diff
@@ -47,10 +47,13 @@ script:
     if [[ "$JOB" != "build_doc" ]]; then exit 0; fi;
     if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then exit 0; fi;
     if [[ "$TRAVIS_BRANCH" != "develop" ]]; then exit 0; fi;
+    # The documentation build fails on Travis machine. Use paddlepaddle.org docker image to have a consistant build environment.
+    docker run -it -v `pwd`:/visualdl --entrypoint "/bin/bash" paddlepaddle/paddlepaddle.org:latest -c 'cd /visualdl; mkdir build; cd build; cmake -DWITH_DOC=ON ..; make -j `nproc` visualdl_docs_en visualdl_docs_cn'
```

Comment:
might make sense at one point to move this script to something like build_doc.sh in visualdl